init/fapolicyd.service

```diff
@@ -1,6 +1,9 @@
+# You should manage this file with systemctl edit utility and not manually
+
 [Unit]
 Description=File Access Policy Daemon
 DefaultDependencies=no
+# Append nss-user-lookup.target to the following line if rules need user/group lookup
```

Comment:
I would prefer a drop-in:
~~~
# If rules need user/group lookup, create a drop-in to delay the startup after NSS lookup is available:
#   # mkdir -p /etc/systemd/system/fapolicyd.service.d
#   # echo -e "[Unit]\nAfter=nss-user-lookup.target" > /etc/systemd/system/fapolicyd.service.d/nss-lookup.conf
#   # systemctl daemon-reload
~~~